notes.txt

```diff
@@ -0,0 +1 @@
+Bo's branch for login.
```

Comment:
What is the purpose of this file? If you want to include a readme for other developers I suggest using markdown https://guides.github.com/features/mastering-markdown/  .md files are supported by github - README.md will be displayed by default in the lower section of the page after opening repository/directory.